src/views/SecondAssessment.vue

```diff
@@ -0,0 +1,113 @@
+<script setup lang="ts">
+import { ref, onMounted } from 'vue'
+import { useRolesStore } from '@/stores/roles'
+import { ElSlider, ElButton } from 'element-plus'
+import type { Role } from '@/domain/Role'
+import type { Knowledge } from '@/domain/Knowledge'
+import { quizAnswer } from '@/stores/roles/state'
+import { ErrorMessage, SuccessMessage } from '@/services/messages'
+
+const roleId = ref('1')
+const rolesStore = useRolesStore()
+const role = ref<Role>({ name: '', knowledges: [], levels: [] })
+const knowledges = ref<Knowledge[]>([])
+
+onMounted(async () => {
+  role.value = await rolesStore.getRoleById(roleId.value)
+  knowledges.value = role.value.knowledges ?? []
```

Comment:
The knowledges should be here, mandatorily. Maybe, you don't even need knowledges variable here
```suggestion
  knowledges.value = role.value.knowledges
```